docs/deployment/version_skew_policy.md

```diff
@@ -0,0 +1,70 @@
+# Version Skew Policy
+
+This document describes the maximum version skew supported between various Gardener components.
+
+## Supported Gardener Versions
+
+Gardener versions are expressed as `x.y.z`, where `x` is the major version, `y` is the minor version, and `z` is the patch version, following Semantic Versioning terminology.
+
+The Gardener project maintains release branches for the most recent three minor releases.
+
+Applicable fixes, including security fixes, may be backported to those three release branches, depending on severity and feasibility.
+Patch releases are cut from those branches at a regular cadence, plus additional urgent releases, when required.
```

Comment:
```suggestion
Applicable fixes, including security fixes, maybe backported to those three release branches, depending on severity and feasibility.
Patch releases are cut from those branches at a regular cadence, plus additional urgent releases when required.
```

Reply:
While I removed the `,`, the `may be` is intended (`maybe` means something different).